src/communication/bluetooth/Bluetooth.cpp

```diff
@@ -104,6 +104,7 @@ void Bluetooth::start(BluetoothHandler *bluetoothHandler) {
   BLEDevice::setSecurityCallbacks(new CustomBLESecurity());
 
   bleServer = BLEDevice::createServer();
+  Bluetooth::getBleServer()->getAdvertising()->addServiceUUID(SERVICE_UUID);
```

Comment:
`bleServer->getAdvertising()->addServiceUUID(SERVICE_UUID);`

This work in the same way but faster xD